tractor/_spawn.py

```diff
@@ -153,6 +156,38 @@ async def cancel_on_completion(
         await portal.cancel_actor()
 
 
+@asynccontextmanager
+async def run_in_process(async_fn, *args, **kwargs):
```

Comment:
To make this simpler why don't we just make the signature of `run_in_proc()` something like:

```python
async def run_in_process(subactor, async_fn, *args, **kwargs):
    debug_args = list(subactor.uid)
    ...
```

I do think it would be neat to accept these over `sys.argv` in `_child.py` and verify just as a sanity check.
It will for sure help with debugging using `pstree` (or whatever).

Heck, maybe it actually makes for sense to do this then passing the `Actor` type at all?
In theory we can avoid passing any complex types whatsoever as long as `Actor._async_main()` receives the appropriate data that `Actor.__init__()` normally does, :thinking:.

Not sure if that's overthinking it.
I wonder if that means we could avoid `cloudpickle` entirely?
We can maybe just use `msgpack` in that case?

Reply:
I think at the least in terms of dropping `cloudpickle` we should make an issue and tackle it later.